src/main/scala/ing/wbaa/druid/definitions/QueryContext.scala

```diff
@@ -0,0 +1,69 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package ing.wbaa.druid.definitions
+
+object QueryContext {
+
+  type QueryContextParam = String
+  type QueryContextValue = String
+
+  // the following apply only to all queries
+  final val Timeout                      = "timeout"
+  final val Priority                     = "priority"
+  final val QueryId                      = "queryId"
+  final val UseCache                     = "useCache"
+  final val PopulateCache                = "populateCache"
+  final val UseResultLevelCache          = "useResultLevelCache"
+  final val PopulateResultLevelCache     = "populateResultLevelCache"
+  final val BySegment                    = "bySegment"
+  final val Finalize                     = "finalize"
+  final val ChunkPeriod                  = "chunkPeriod"
```

Comment:
Maybe add a deprecation method here as well? 

https://github.com/apache/incubator-druid/blob/master/processing/src/main/java/org/apache/druid/query/QueryContexts.java#L42

Appearently you can also add `@deprecated` to `val`'s:
```
MacBook-Pro-van-Fokko:incubator-druid fokkodriesprong$ scala
Welcome to Scala 2.13.0 (Java HotSpot(TM) 64-Bit Server VM, Java 1.8.0_172).
Type in expressions for evaluation. Or try :help.

scala>  @deprecated("this val will be removed")
     | val vo = "lid"
vo: String = lid

scala> vo
       ^
       warning: value vo is deprecated: this val will be removed
res0: String = lid
```

Reply:
ok now `ChunkPeriod` is marked as deprecated